spec/latest/core/core.jsonld

```diff
@@ -431,6 +432,16 @@
       "@id": "hydra:template",
       "@type": "rdf:Property",
       "label": "template",
+      "comment": "A templated string with placeholders. The literal's datatype indicates the template syntax; if not specified, hydra:rfc6570Template is assumed.",
+      "seeAlso": "hydra:rfc6570Template",
+      "domain": "hydra:IriTemplate",
+      "range": "rdfs:Literal",

```

Comment:
Not necessarily I’d say. What can *machines* do in practice if we add this range that they can't otherwise?